docs/pbc/all/payment-service-providers/unzer/install-unzer/integrate-unzer.md

```diff
@@ -352,7 +608,7 @@ class UnzerGuiDependencyProvider extends SprykerUnzerGuiDependencyProvider
 
 </details>
 
-9. Add `StepHandler` plugins to `CheckoutPageDependencyProvider`:
+7. Add `StepHandler` plugins to `CheckoutPageDependencyProvider`:
 
 <details>
 <summary markdown='span'>src/Pyz/Zed/UnzerGui/UnzerGuiDependencyProvider.php</summary>
```

Comment:
please check this block
path and namespace do not match each other
class definition is missing